R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
I see. But as you mentioned, we don't know how to trigger the write path on the R side, because both bigint and double are `numeric`. I think we can just remove the test in the R side.